src/bindings/js/node/tests/unit/infer_request.test.js

```diff
@@ -300,3 +300,47 @@ describe('ov.InferRequest tests', () => {
     });
   });
 });
+
+describe('ov.InferRequest tests with missing outputs names', () => {
+  const modelV3Small = testModels.modelV3Small;
```

Comment:
```suggestion
  const { modelV3Small } = testModels;
```

Reply:
Thanks, done. I think we need to add the rule in out lint config.